xrayvision/tests/test_imaging.py

```diff
@@ -30,27 +31,31 @@ def uv():
 @pytest.mark.parametrize("pixel_size", [(0.5), (1), (2)])
 def test_psf_to_image(pixel_size, uv):
     u, v = uv
-    img = np.zeros((65, 65)) * (apu.ph / apu.arcsec**2)
-    img[32, 32] = 1.0 * (apu.ph / apu.arcsec**2)
-    obs_vis = dft_map(img, u=u, v=v, pixel_size=[2.0, 2.0] * apu.arcsec)
+    img = np.zeros((65, 65)) * (apu.ph / apu.cm**2)
+    img[32, 32] = 1.0 * (apu.ph / apu.cm**2)
```

Comment:
Aren't the units of a map photons (or counts) s^-1 cm^-2 keV^-1 arcsec^-2? I think the normalization by arcsec^2 is needed given pixel_size of dft_map is in apu.arcsec / apu.pix. The normalization by s and keV can be neglected if not considered here

Reply:
The units here don't really matter but I should double check that the result of the `idft` have what ever units the input visibility have with an additional arcsec^-2

Reply:
Ok thanks!

Reply:
So currently the underlying code `dft`, `idft` don't really do anything with the units , specifically `idft(vis, ....)` will return something with the units of `vis` and doesn't add the `1/arcsec^2` it really should. Similarly `dft(an_array, ...)` doesn't check if an_array has `1/arcsec^2` in its units and just return `vis` with the same units as `an_array`.